### ISSUE TITLE
[Refactor] Floating label not changing direction

### ISSUE THREAD

**Issue by daniel-ilan** [hasadna/anyway-newsflash-infographics#704]
Blocked by #843 

1. Go to home page
2. Log in
3. Click "פרטי משתמש" while on Hebrew language
  - see the floating label is floating on the top left side of the input - it should change direction to the right

![image](https://user-images.githubusercontent.com/37712687/125154645-a0c1cf00-e163-11eb-8ac8-f34e987648a5.png)

**Comment by daniel-ilan** [comment]
From what I can tell we are missing a metirial-ui dependency when changing direction.
According to their [documentation ](https://material-ui.com/guides/right-to-left/#3-jss-rtl) we should install jss-rtl plugin and initiate it after the theme

Also see this [stack-overflow discussion](https://stackoverflow.com/questions/62799638/material-ui-textfield-not-affected-with-the-rtl-direction) about the same

**Comment by yuvalbl** [comment]
Reopened - see my comment on #740 
will be postponed till after we upgrade to MUI (v5)  - #790
